docs/source/partial-transformers/partial-transformers.rst

```diff
@@ -166,6 +166,54 @@ You can expect that basic functionality of chimney's ``Transformer`` either work
 fashion in ``PartialTransformer``\s, or have some counterparty methods in the API
 (usually with the `Partial` prefix or suffix in the name).
 
+Preferring total or partial transformer
+---------------------------------------
+
+Under normal circumstances when fields or sealed hierarchy subtypes should be converted, Chimney should be able
+to generate code on its own. If it cannot you could provide it with derivation settings or implicit transformer handling
+problematic fields/subtypes.
+
+But which implicit should be taken in this situation?
+
+.. code-block:: scala
+
+  import io.scalaland.chimney._
+  import io.scalaland.chimney.dsl._
+
+  case class UserAPI(credits: String)
+  case class User(credits: Int)
+
+  implicit val unsafeStringToInt: Transformer[String, Int] = _.toInt
+
+  implicit val parseStringToInt: PartialTransformer[String, Int] =
+    PartialTransformer[String, Int](str => partial.Result.fromCatching(str.toInt).map(_ * 2))
+
+  UserAPI("10").transformIntoPartial[User].asOption // Some(User(10)) or Some(User(20)) ?
```

Comment:
We should explicitly mention that this example ends up with a compilation error.

Reply:
done